pkg/endpoint/policy.go

```diff
@@ -349,13 +350,18 @@ func (e *Endpoint) regenerateL3Policy(owner Owner) (bool, error) {
 		ctx.Trace = policy.TRACE_ENABLED
 	}
 	newL3policy := repo.ResolveL3Policy(&ctx)
+	// Perform the validation on the new policy
+	err := l3PolicyValidation(newL3policy)
+	change = err == nil
```

Comment:
It's not quite the right variable name, this doesn't mean a `change`, it means it is valid.

Reply:
Was thinking it could be reused in the future, but changing it to `valid` for now.